### PR TITLE
First eight tasks

### PR DESCRIPTION
**Pull request by wazzuper**
В это ветке чуть больше половины заданий по чек-листу:

- Настройка ruby, rails, bundler и postgres

- Создание репозитория, приложения и опубликование кода здесь

- Установка и настройка RSpec, capybara; добавление первого теста (нерабочего, для будущего "Hello World")

- Настройка .env, установка гема dotenv-rails

- Установка и настройка rubocop и reek

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require: rubocop-rspec
```
Comment (atatarenko) [inline .rubocop.yml:1]: My advice is to use more strict rules for the Rubocop. This is one of the easiest tricks helping you write the clean code. Moreover, it doesn't really matter which rules you have. The point in having it.

Take a look at the [config](https://gist.github.com/atatarenko/c72a6e01cde7a96ee10527f49a91bbc5) from the current project. It isn't perfect, really, but more strict.

File: Gemfile [hunk 2]
```diff
@@ -39,6 +39,11 @@ group :development, :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
+
+  gem 'rspec-rails', '~> 3.7'
+  gem 'factory_bot_rails'
```
Comment (atatarenko) [inline Gemfile:43]: Wow, cool 👍
Actually, I'm an old-fashioned guy loving `factory girl`.
Just out of curiosity, have you used the `factory_bot_rails ` before? And why it's better than factory girl?

File: .gitignore [hunk 3]
```diff
@@ -4,8 +4,8 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
-# Secrets
-/config/secrets.yml
+# Ignore all .env files
+/.env*
```
Comment (atatarenko) [inline .gitignore:8]: Let's add `public/packs` and `/public/packs-test` to ignore file too.

File: Gemfile (hunk 2 above)
Comment (wazzuper) [reply Gemfile:43]: Yes, I've used it in this little project https://github.com/wazzuper/blogarium

All I know about difference between ```factory girl``` and ```factory_bot_rails``` - it's their names, this is the same gem, but with a different name :)

File: .gitignore [hunk 4]
```diff
@@ -4,8 +4,12 @@
...
+
+# Ignore all /public/packs and /public/packs-test files
+/public/packs/*
+/public/packs-test/*
```
Comment (atatarenko) [inline .gitignore]: Try:
```
public/packs
public/packs-test
```

These two guys mustn't go into repo :) Every developer should have their own copy of the packs

P.S.
+40K seems to much, doesn't it :) ?
screen shot 2018-02-22 at 10 24 16 pm

**Pull request merged**